static/js/components/tiles/line_tile.tsx

```diff
@@ -111,6 +107,14 @@ export function LineTile(props: LineTilePropType): JSX.Element {
   );
 }
 
+export function getReplacementStrings(
```

Comment:
Add comment on what the replacement string is used for?

Reply:
done